doc/api/process.md

```diff
@@ -1711,15 +1711,15 @@ the script name. These options are useful in order to spawn child processes with
 the same execution environment as the parent.
 
 ```bash
-node --harmony script.js --version
+node --input-type=module --eval 'console.log("foo")' script.js --version
```

Comment:
It's quite weird to have `--eval` + positional arguments, in particular it's not obvious (to me anyway) if `script.js` would be executed or not without trying the command.

Reply:
Would you rather it be switched?

```bash
node --input-type module --eval='console.log("foo")' script.js --version
```

What the flags are nor their values _really_ matter here. The point here is to demonstrate how `=` vs ` ` affect the items in `process.execArg`. Happy to switch them to something more appropriate to `=` vs ` `. The flags that actually brought to light the deficiency in the existing example were:

```bash
node --watch-path=./foo --require ./bar.js script.js --version
```

`script.js` will be executed 😉 And the example being this way is important because it shows `--version` will not be present in `process.execArg` (contrary to what some users may expect).

Reply:
```suggest
node --watch-path=./foo --require ./bar.js script.js --version
```